actions/threads.tsx

```diff
@@ -41,6 +32,7 @@ export async function getThreads() {
   try {
     threadDirs = await fs.readdir(threadsDir);
   } catch (e) {
+    console.error('Failed to read threads directory', e);
```

Comment:
The first time I started the app, I got this error:

```
Failed to read threads directory Error: ENOENT: no such file or directory, scandir '/Users/nick/Library/Application Support/acorn/Acorn/threads'
    at async readdir (node:internal/fs/promises:948:18)
    at async m (/Users/nick/projects/gptscript-ai/desktop/electron-dist/mac-arm64/Acorn.app/Contents/Resources/app.asar/.next/server/chunks/500.js:6:22272)
    at async /Users/nick/projects/gptscript-ai/desktop/electron-dist/mac-arm64/Acorn.app/Contents/Resources/app.asar/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:16:408
    at async rP (/Users/nick/projects/gptscript-ai/desktop/electron-dist/mac-arm64/Acorn.app/Contents/Resources/app.asar/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:15:8043)
    at async r5 (/Users/nick/projects/gptscript-ai/desktop/electron-dist/mac-arm64/Acorn.app/Contents/Resources/app.asar/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:18:1139)
    at async doRender (/Users/nick/projects/gptscript-ai/desktop/electron-dist/mac-arm64/Acorn.app/Contents/Resources/app.asar/node_modules/next/dist/server/base-server.js:1433:30)
    at async cacheEntry.responseCache.get.routeKind (/Users/nick/projects/gptscript-ai/desktop/electron-dist/mac-arm64/Acorn.app/Contents/Resources/app.asar/node_modules/next/dist/server/base-server.js:1594:28)
    at async NextNodeServer.renderToResponseWithComponentsImpl (/Users/nick/projects/gptscript-ai/desktop/electron-dist/mac-arm64/Acorn.app/Contents/Resources/app.asar/node_modules/next/dist/server/base-server.js:1502:28)
    at async NextNodeServer.renderPageComponent (/Users/nick/projects/gptscript-ai/desktop/electron-dist/mac-arm64/Acorn.app/Contents/Resources/app.asar/node_modules/next/dist/server/base-server.js:1919:24)
    at async NextNodeServer.renderToResponseImpl (/Users/nick/projects/gptscript-ai/desktop/electron-dist/mac-arm64/Acorn.app/Contents/Resources/app.asar/node_modules/next/dist/server/base-server.js:1957:32) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'scandir',
  path: '/Users/nick/Library/Application Support/acorn/Acorn/threads'
}
```

It's not really a big deal since it didn't impact usability and I didn't get it on the second run, but does pollute the logs a bit. 

It's probably worth checking the exception for `ENOENT` here to avoid logging the error.

Reply:
Yeah good call. I will fix that